src/AzSK.ADO/Framework/Configurations/SVT/ControlSettings.json

```diff
@@ -291,29 +291,50 @@
         "Environment",
         "SolutionName"
     ],
-    "RestrictedBroaderGroupsForBuild" : {
-      "Contributors":[
+    "RestrictedBroaderGroupsForBuild": {
```

Comment:
Please add all the settings in the oss-config policy file, which is present in the src folder.
oss-config-preview/1.0.0 
oss-config-staging/1.0.0
oss-config/1.0.0